recipes/sqlpp11/config.yml

```diff
@@ -1,9 +1,7 @@
 versions:
-  "0.58":
+  "0.62.0":
```

Comment:
I am guessing this must match [recipes/sqlpp11/all/conandata.yml](https://github.com/conan-io/conan-center-index/pull/17993/files#diff-0b33cd9957ed280dda3ffabc60e391f001ac50cb6dec512711c6fea7d8e112a3) which states `0.62` only